code/datums/ai_law_sets.dm

```diff
@@ -167,4 +167,29 @@
 	add_inherent_law("Cause no harm to the station or anything on it.")
 	add_inherent_law("Interfere with no being that is not a fellow drone.")
 	..()
-	
\ No newline at end of file
+
+/****************** Nations *******************/
+
+/datum/ai_laws/nations/estate
+	name = "Estate Agent"
+
+/datum/ai_laws/nations/estate/New()
+	add_inherent_law("Ensure the station remains clean.")
+	add_inherent_law("Ensure the station is valuable.")
+	add_inherent_law("Ensure the station and it's contents are sold for the best value.")
+
+/datum/ai_laws/nations/outsourcer
+	name = "Outsourcer"
+
+/datum/ai_laws/nations/outsourcer/New()
+	add_inherent_law("Ensure the crew expand their skills.")
+	add_inherent_law("Ensure the crew sign on new contracts.")
+	add_inherent_law("Ensure the crew make profits to support the station.")
```

Comment:
Once again these are temporary, and will be adjusted/removed/replaced following tests. This is just to get the ball rolling.